docs/docsite/rst/community/development_process.rst

```diff
@@ -37,104 +37,14 @@ If you want to contribute a feature or fix a bug in ``ansible-core`` or in a col
 Here's an overview of the PR lifecycle:
 
 * Contributor opens a PR (always against the ``devel`` branch)
-* Ansibot reviews the PR
-* Ansibot assigns labels
-* Ansibot pings maintainers
+* `Ansibot <https://github.com/ansible/ansibotmini#ansibotmini>`_ triages the PR
```

Comment:
How about adjusting this to:
```suggestion
* ansible-core uses `Ansibot <https://github.com/ansible/ansibotmini#ansibotmini>`_ to triage the PR.
  Some collection repositories use `Ansibullbot <https://github.com/ansible-community/collection_bot/blob/main/ISSUE_HELP.md>`_ to triage the PR. For most collections, this is done manually or by other means.
```

(The previous and the next line should also be changed, but I guess that's for another PR since this one is about the bot.)